qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -1525,12 +1551,32 @@ def _add_parameter_value_from_conf(
         self.add_parameter_value(param_val, *key, update_inst_map=False)
 
     @classmethod
-    def load(cls, files: List[str]) -> "Calibrations":
+    @deprecate_arg(
+        name="files",
+        new_alias="file_path",
+        since="0.6",
+        package_name="qiskit-experiments",
+    )
```

Comment:
Why do we need a deprecation warning here? This is unclear to me. If you are loading form a JSON, as you do here, I do not see why we should deprecate this function. Or alternatively, I don't see why you want to implement a function to deprecate it.

Reply:
This is deprecation for arg name. `files` indicates list of paths assuming the split csv files. This is just renamed to `file_path` indicating a single source file of json.